### PR TITLE
[AP-29] Add cdc service and bonus metadata

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR adds the CDC special service and the CDC to the available bonus metadata.
Note that the bonus metadata is filled with fake data and is at the moment not created in production.

### CODE REVIEW

File: assets/bonus_available/bonus_available_v2.json [hunk 1]
```diff
@@ -106,5 +106,31 @@
     "cover": "https://assets.cdn.io.italia.it/bonus/cgn/logo/logo-cgn.png",
     "sponsorship_description": "PCM - Dipartimento per le Politiche Giovanili e il Servizio Civile Universale",
     "sponsorship_cover": "https://assets.cdn.io.italia.it/bonus/cgn/logo/logo-cgn.png"
+  },
+  {
+    "id_type": 4,
+    "it": {
+      "name": "Carta Della Cultura ",
```
Comment (Undermaken) [inline assets/bonus_available/bonus_available_v2.json:113]:
```suggestion
      "name": "Carta Della Cultura",
```

File: assets/bonus_available/bonus_available_v2.json [hunk 2]
```diff
@@ -106,5 +106,31 @@
...
+      "description": "Carta Della Cultura (CDC) è l’incentivo per i giovani che favorisce la partecipazione ad attività culturali, sportive e ricreative, su tutto il territorio nazionale",
+      "subtitle": "Carta Della Cultura (CDC) è l’incentivo per i giovani che favorisce la partecipazione ad attività culturali, sportive e ricreative, su tutto il territorio nazionale",
+      "title": "Attiva la Carta Della Cultura",
+      "content": ""
+    },
+    "en": {
+      "name": "Carta Giovani Nazionale",
```
Comment (Undermaken) [inline assets/bonus_available/bonus_available_v2.json:120]:
```suggestion
      "name": "Carta Della Cultura",
```

File: src/payloads/services/special.ts [hunk 3]
```diff
@@ -53,14 +53,34 @@ const withCgnService = (
...
 // list of tuple where the first element is a flag indicating if the relative service should be included
 // the second element is the service factory
 const specialServicesFactory: ReadonlyArray<readonly [
   boolean,
   (organizationFiscalCode: OrganizationFiscalCode) => ServicePublic
 ]> = [
   [ioDevServerConfig.services.includeSiciliaVola, withSiciliaVolaService],
-  [ioDevServerConfig.services.includeCgn, withCgnService]
+  [ioDevServerConfig.services.includeCgn, withCgnService],
+  [true, withCdcService]
```
Comment (Undermaken) [inline src/payloads/services/special.ts:83]:
```suggestion
  [true, withCdcService]
```
Could be a good idea update the config by including a flag for CDC service too?

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: assets/bonus_available/bonus_available_v2.json (hunk 2 above)
Comment (debiff) [reply assets/bonus_available/bonus_available_v2.json]: Thank you. Fixed in [272d71c](https://github.com/pagopa/io-dev-api-server/pull/171/commits/272d71ca6d39e0d5ee6086481abd8665ec02f4b4)

**Pull request merged**